module/renderer/src/commonTest/kotlin/org/cru/godtools/shared/renderer/content/BaseRenderContentTest.kt

```diff
@@ -75,4 +76,66 @@ abstract class BaseRenderContentTest {
         }
     }
     // endregion Clickable
+
+    @Test fun `IsGone`() = runComposeUiTest {
+        state.setVar("gone", listOf("value"))
+
+        setContent {
+            RenderContentStack(
+                listOf(testModel),
+                state = state,
+            )
+        }
+
+        testScope.runTest {
+            turbineScope {
+                assertTrue(testModel.isGone(state = state))
+            }
+        }
```

Comment:
what I'm thinking for this you will want something like this:

```kotlin
onModelNode().assertExists()

state.setVar("gone", listOf("value"))
onModelNode().assertDoesNotExist()
```

Reply:
and you can potentially clear the `gone` var and make sure the node is present again